templates/todo/web/react-fluent/package.json

```diff
@@ -20,8 +20,8 @@
     "build": "tsc -b .",
     "pretest": "npm run envconfig",
     "test": "if-node-version '>= 17' && react-scripts --openssl-legacy-provider test || react-scripts test",
-    "prepackage": "npm run envconfig",
     "package": "if-node-version '>= 17' && react-scripts --openssl-legacy-provider build || react-scripts build",
+    "postpackage": "cp ./entrypoint.sh ./build",
```

Comment:
This ensures our custom `entrypoint.sh` is available on the webserver for custom command.

Reply:
Makes sense. There isn't a different way to declare this as an `asset` file, i'm guessing?